docs/source/timing-analysis.rst

```diff
@@ -82,12 +94,34 @@ Then you can install tlsfuzzer dependencies to speed-up the test execution:
 
 .. code:: bash
 
-   dnf install python3 python3-devel tcpdump gmp-devel swig mpfr-devel \
-   libmpc openssl-devel make gcc gcc-c++ git libmpc-devel python3-six
+   dnf install -y python3 python3-devel tcpdump gmp-devel swig mpfr-devel \
+   libmpc openssl-devel make gcc gcc-c++ git libmpc-devel python3-six \
+   meson openblas-devel gcc-gfortran lapack-devel zlib-devel \
+   libtiff-devel libjpeg-devel openjpeg2-devel zlib-devel freetype-devel \
```

Comment:
zlib-devel is duplicated

Reply:
actually, given the complexity of getting them all installed, I'll probably provide both commands, as it affects only people running tests on platforms other than x86_64 and aarch64

Reply:
fixed